packages/flutter/lib/fix_data.yaml

```diff
@@ -17,6 +17,65 @@
 
 version: 1
 transforms:
+  - title: 'Remove BoxShape.rectangle'
+    date: 2022-07-27
+    element:
+      uris: [ 'widgets.dart', 'material.dart', 'cupertino.dart' ]
+      inClass: 'BoxDecoration'
+      constructor: ''
```

Comment:
As this one if for the constructor, there should also be a fix for the `copyWith` method.